README.md

```diff
@@ -120,7 +120,7 @@ velero restore create --from-backup sched-20190513104034 --restore-volumes=true
 
 ## Compatibility matrix
 
-|     Image           |    Codebase     |  Heptio Velero v0.10.0  | Heptio Velero v0.11.0 |
+|     Image           |    Codebase     |  Velero v0.10.0  | Velero v0.11.0 |
```

Comment:
With this merge, 'ci' will be supporting Velero 1.0.0-rc.1

Reply:
yes